packages/gcc/8.1.0/0019-ARC-Add-multilib-support-for-linux-targets.patch

```diff
@@ -0,0 +1,64 @@
+From 3d9f0e2c30020c21f2cf4ef30e839fdb08ad7cd3 Mon Sep 17 00:00:00 2001
+From: Alexey Brodkin <abrodkin@synopsys.com>
+Date: Wed, 16 May 2018 13:29:58 +0300
+Subject: [PATCH] ARC: Add multilib support for linux targets
```

Comment:
Is it being submitted upstream? If so, can you add PR number for reference?

Reply:
Not yet. I just sent it for internal review and will shortly send it to GCC's mailing list.
Let's wait then before it gets reviewed and merged in GCC's trunk - I'll update this pull-request then.